bash/solutii/marius.spinu/Man/man.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+
+    if [[ $# -ne 2 ]]; then
+        echo "Insufficient parameters"
+        exit 0
+    fi
+
+    if [[ $(which "$1") == '' ]]; then
```

Comment:
It is better to use the EXIT STATUS of which for checking if a command is present or not. If the exit value is 1 then it is not present.